### PR TITLE
Plugin modernization

### LINKED ISSUE #5: Update dependencies (needs some rewrite)
eslint has changed how rules are defined in plugins and how tests are run, so updating will need a bit of rewriting.

I'm unlikely to spend a lot of time on this, but happy to help someone do this with my support and guidance.

### PR DESCRIPTION
**Pull request by alexlafroscia**
Closes #5

### CODE REVIEW

File: lib/rules/no-wildcard-postmessage.js [hunk 1]
```diff
@@ -9,32 +10,56 @@
...
+          funcName = node.callee.property.name;
         } else {
+          // anonymous function
           return;
         }
 
+        if (funcName === "postMessage") {
+          const [, originNode] = node.arguments;
+
+          if (isWildcard(originNode)) {
```
Comment (mozfreddyb) [inline lib/rules/no-wildcard-postmessage.js:58]: Not sure this is worth the `isWildcard` and `isLiteral` indirection, when you're using both functions just once

**Review by mozfreddyb — COMMENTED** [review]
Looks great, thank you so much for updating the library
Do you mind changing authorship in files (`@author ..`) and npm package.json to "eslint-plugin-no-wildcard-postmessage contributors" or something?

**Comment by mozfreddyb** [comment]
@alexlafroscia can I count you in on the two remaining minor changes? Otherwise I'll merge and do those edits myself

**Comment by alexlafroscia** [comment]
Sorry, yeah — I can make those tweaks myself today.

**Review by mozfreddyb — APPROVED** [review]

**Comment by mozfreddyb** [comment]
thank you :star_struck:

**Comment by mozfreddyb** [comment]
published as 0.2.0 at https://www.npmjs.com/package/eslint-plugin-no-wildcard-postmessage

**Pull request merged**